docs/src/man/guide.md

```diff
@@ -2,134 +2,123 @@
+Go back to the Pkg REPL and generate a new package:
 
-```
-Example/
-├── src/
-│   └── Example.jl
-...
+```julia
+(@v1.5) pkg> generate Example
+ Generating  project Example:
+    Example/Project.toml
+    Example/src/Example.jl
 ```
 
-Note that the `...` just represent unimportant files and folders.
+By default the generated Example.jl file contains a `greet()` function. We're going to add a
+docstring to it:
 
-We must decide on a location where we'd like to store the documentation for this package.
-It's recommended to use a folder named `docs/` in the toplevel of the package, like so
+```julia
+module Example
 
-```
-Example/
-├── docs/
-│   └── ...
-├── src/
-│   └── Example.jl
-...
-```
+"""
+    greet()
 
-Inside the `docs/` folder we need to add two things. A source folder which will contain the
-markdown files that will be used to build the finished document and a Julia script that will
-be used to control the build process. The following names are recommended
+Prints "Hello world!"
+"""
+greet() = print("Hello World!")
 
-```
-docs/
-├── src/
-└── make.jl
+end # module
 ```
 
+Now create a `docs` folder within the package with the following structure:
 
-## Building an Empty Document
-
-With our `docs/` directory now setup we're going to build our first document. It'll just be
-a single empty file at the moment, but we'll be adding to it later on.
+```
+Example/
+├── docs
+│   ├── make.jl
+│   └── src
+│       └── index.md
+├── Project.toml
+└── src
+    └── Example.jl
+```
 
-Add the following to your `make.jl` file
+The `make.jl` file should contain:
 
 ```julia
 using Documenter, Example
 
 makedocs(sitename="My Documentation")
 ```
 
-This assumes you've installed Documenter as discussed in [Installation](@ref) and that your
-`Example.jl` package can be found by Julia.
+and the `index.md` look as follows:
 
-!!! note
-
-    If your source directory is not accessible through Julia's LOAD_PATH, you might wish to
-    add the following line at the top of make.jl
+````
+# Example.jl Documentation
 
-    ```julia
-    push!(LOAD_PATH,"../src/")
-    ```
+```@docs
+Example.greet()
+```
+````
 
-Now add an `index.md` file to the `src/` directory.
+Since Documenter is a development tool and not a true dependency of your package, create
+a fresh environment in the `docs/` folder, and add Documenter and the Example package
+itself as dependencies:
 
-!!! note
-    If you use Documenter's default HTML output the name `index.md` is mandatory.
-    This file will be the main page of the rendered HTML documentation.
+```julia
+(@v1.5) pkg> activate Example/docs
+ Activating new environment at `~/Documents/projects/Example/docs/Project.toml`
 
-Leave the newly added file empty and then run the following command from the `docs/` directory
+(docs) pkg> add Documenter
 
-```sh
-$ julia make.jl
+(docs) pkg> dev Example
+[ Info: Resolving package identifier `Example` as a directory at `~/Documents/projects/Example`.
```

Comment:
This recommendation could be problematic, because it could clone a new package into `~/.julia/dev/Example.jl`, if you don't happen to be in the right directory? Perhaps better to use a relative path (`pkg> dev .` if in package root, or `pkg> dev ..` if under `docs/`) ?